src/components/UserPortal/ChatRoom/ChatRoom.spec.tsx

```
@@ -77,6 +122,11 @@ import {
   DELETE_CHAT_MESSAGE,
 } from 'GraphQl/Mutations/OrganizationMutations';
 
+export const FIXED_UTC = new Date(Date.UTC(2023, 0, 1)).toISOString();
```

Biome (2.4.6)

[error] Do not export from a test file.

(lint/suspicious/noExportsInTest)